server/handlers/result_handlers.py

```diff
@@ -0,0 +1,77 @@
+from typing import Optional
+import logging
+import json
+from abc import abstractmethod
+from pydantic import computed_field, ConfigDict, model_validator
+
+from adala.utils.registry import BaseModelInRegistry
+from label_studio_sdk import Client
+
+
+logger = logging.getLogger(__name__)
+
+
+class ResultHandler(BaseModelInRegistry):
+    @abstractmethod
+    def __call__(self, batch):
```

Comment:
nit: mind naming this param `result_batch`? I know it's a little verbose, but imo when building an interface for users/developers to extend, it's important to make it really obvious how to implement the interface